src/types/currency.ts

```diff
@@ -1,93 +1,133 @@
-import { makeClass } from "../utils/make-class";
-const _ = require("lodash");
-const { slice } = require("../utils/bytes-utils");
-const { Hash160 } = require("./hash-160");
+import { Hash160 } from "./hash-160";
 const ISO_REGEX = /^[A-Z0-9]{3}$/;
 const HEX_REGEX = /^[A-F0-9]{40}$/;
 
-function isoToBytes(iso) {
-  const bytes = new Uint8Array(20);
+/**
+ * Convert an iso code to a currency bytes representation
+ */
+function isoToBytes(iso: string): Buffer {
+  const bytes = Buffer.alloc(20);
   if (iso !== "XRP") {
     const isoBytes = iso.split("").map((c) => c.charCodeAt(0));
     bytes.set(isoBytes, 12);
   }
   return bytes;
 }
 
-function isISOCode(val) {
-  return val.length === 3; // ISO_REGEX.test(val);
+/**
+ * Tests if iso is a valid iso code
+ */
+function isISOCode(iso: string): boolean {
```

Comment:
optional: Like `ID` vs `Id` above, would recommend `isIsoCode` here unless it breaks a public API.